core/src/main/java/de/neofonie/styla/core/utils/SeoUtils.java

```diff
@@ -51,95 +60,94 @@ private static void writeMetaTags(final Seo seo, final Resource resource, final
+                    "og:image:height".equalsIgnoreCase(tag.getAttributes().get(ATTRIBUTE_PROPERTY).getAsString())) {
+                properties.put("stylaOgImageHeight", tag.getAttributes().get(ATTRIBUTE_CONTENT).getAsString());
+            }
 
-            boolean isProperty = StringUtils.isNotEmpty(metaTag.getProperty());
-            properties.put("tagLabel", isProperty ? metaTag.getProperty() : metaTag.getName());
-            properties.put("tagValue", metaTag.getContent());
-            properties.put("isProperty", isProperty);
+            // twitter:title
+            if (TAG_META.equalsIgnoreCase(tag.getTag()) && tag.getAttributes() != null &&
+                    tag.getAttributes().has(ATTRIBUTE_PROPERTY) &&
+                    "twitter:title".equalsIgnoreCase(tag.getAttributes().get(ATTRIBUTE_PROPERTY).getAsString())) {
+                properties.put("stylaTwitterTitle", tag.getAttributes().get(ATTRIBUTE_CONTENT).getAsString());
+            }
 
-            ResourceUtil.getOrCreateResource(resourceResolver, (metaTagsResource.getPath() + "/item" + String.valueOf(numberOfMetaTags)), properties, null, true);
-        } catch (PersistenceException e) {
-            LOGGER.error("Could not create metatags node for " + contentResource.getPath() + "/metatags", e);
-        }
-    }
+            // twitter:description
+            if (TAG_META.equalsIgnoreCase(tag.getTag()) && tag.getAttributes() != null &&
+                    tag.getAttributes().has(ATTRIBUTE_PROPERTY) &&
+                    "twitter:description".equalsIgnoreCase(tag.getAttributes().get(ATTRIBUTE_PROPERTY).getAsString())) {
+                properties.put("stylaTwitterDescription", tag.getAttributes().get(ATTRIBUTE_CONTENT).getAsString());
+            }
 
-    private static void checkForOpenGraphTags(MetaTagJsonUtils.MetaTag metaTag, Map<String, Object> properties) {
-        if (StringUtils.equals(metaTag.getProperty(), "og:type") && metaTag.getContent() != null) {
-            properties.put("ogType", metaTag.getContent());
-        }
-        if (StringUtils.equals(metaTag.getProperty(), "og:title") && metaTag.getContent() != null) {
-            properties.put("ogTitle", metaTag.getContent());
-        }
-        if (StringUtils.equals(metaTag.getProperty(), "og:url") && metaTag.getContent() != null) {
-            properties.put("ogUrl", metaTag.getContent());
-        }
-        if (StringUtils.equals(metaTag.getProperty(), "og:image") && metaTag.getContent() != null) {
-            properties.put("ogImage", metaTag.getContent());
+            // twitter:image
+            if (TAG_META.equalsIgnoreCase(tag.getTag()) && tag.getAttributes() != null &&
+                    tag.getAttributes().has(ATTRIBUTE_PROPERTY) &&
+                    "twitter:image".equalsIgnoreCase(tag.getAttributes().get(ATTRIBUTE_PROPERTY).getAsString())) {
+                properties.put("stylaTwitterImage", tag.getAttributes().get(ATTRIBUTE_CONTENT).getAsString());

```

Comment:
As mentioned above shouldn't title, description, and next/prev be here as well?

Reply:
Actually description, next and prev seems to be missing ...

Reply:
added description, next and prev links